src/ccache.cpp

```diff
@@ -1652,8 +1652,21 @@ calculate_result_name(Context& ctx,
     }
 
     if (Util::starts_with(args[i], "-specs=")
-        || Util::starts_with(args[i], "--specs=")) {
-      std::string path = args[i].substr(args[i].find('=') + 1);
+        || Util::starts_with(args[i], "--specs=")
+        || args[i] == "-specs"
+        || args[i] == "--specs") {
```

Comment:
A hint to clang-format:

```suggestion
        || (args[i] == "-specs" || args[i] == "--specs")) {
```